recipes/opencv/4.x/conanfile.py

```diff
@@ -50,7 +51,8 @@ class OpenCVConan(ConanFile):
         "with_gtk": True,
         "with_quirc": True,
         "with_cuda": False,
-        "with_cublas": False
+        "with_cublas": False,
+        "dnn": False
```

Comment:
We could turn it on to see it fail?

```suggestion
        "dnn": True
```